modules/admin-ui-frontend/app/scripts/shared/services/restServiceMonitor.js

```diff
@@ -45,6 +48,43 @@ angular.module('adminNg.services')
 
     Monitoring.getActiveMQStats();
     Monitoring.getBasicServiceStats();
+    Monitoring.getVersionStats();
+  };
+
+  Monitoring.getVersionStats = function() {
+    $http.get(MY_VERSION_PATH).then(function(response_my_version) {
+      $http.get(LATEST_VERSION_PATH).then(function(response_latest_version) {
+        Monitoring.populateService(LATEST_VERSION_NAME);
+
+        if (response_latest_version.status === 200 && response_my_version.status === 200
+        && response_my_version.data.consistent) {
+          var my_version = response_my_version.data.version;
+          var latest_version = response_latest_version.data;
+          services.service[LATEST_VERSION_NAME].docs_url =
+            'https://docs.opencast.org/r/' + latest_version[0] + '.x/admin/';
```

Comment:
Testing this, I patched out the endpoint to return "10.0" as the version.  This line generates links to `https://docs.opencast.org/r/1.x/admin/` rather than `https://docs.opencast.org/r/10.x/admin/`

Otherwise looks good and works as expected!

Reply:
You are right. I patched this and it should work now. Thanks!